File: 1. Beginner/1008/1008.cs
using System; 
class URI {
    static void Main(string[] args) { 
        int n = Int32.Parse(Console.ReadLine());
        int m = Int32.Parse(Console.ReadLine());
        float s = float.Parse(Console.ReadLine());
       
        Console.Write("NUMBER = {0}\nSALARY = U$ {1}\n", n, (m *s).ToString("F2"));
    }
}

File: 1. Beginner/1008/1008.go
package main

import (
	"fmt"
)

func main() {
	var n int
	var m, s float64

	fmt.Scanf("%d\n%f\n%f", &n, &m, &s)

	fmt.Printf("NUMBER = %d\nSALARY = U$ %.2f\n", n, m * s)
}